src/parser/parser.yy

```diff
@@ -610,9 +610,13 @@ step_clause
         ifOutOfRange($1, @1);
         $$ = new StepClause($1);
     }
-    | KW_UPTO INTEGER KW_STEPS {
-        ifOutOfRange($2, @2);
-        $$ = new StepClause($2, true);
+    | INTEGER KW_TO INTEGER KW_STEPS {
```

Comment:
In that case, does that mean that if the "upto" statement is used it will return an error? If so, I think we need to be compatible with the syntax "upto".

Reply:
As I known, this grammar `UPTO` never introduced into nebula.

Reply:
> As I known, this feature `UPTO` never introduced into nebula, and no plan to support in later.

ok